src/components/folder-dropzone.js

```diff
@@ -125,6 +142,7 @@ export default class FolderDropzone extends React.Component {
           const paths = getFiles(
             filesAndFoldersMapToArray(filesAndFolders)
           ).map(file => file.id);
+          this.handleZips(paths);
```

Comment:
Eviter au maximum l'usage de this. Mieux vaut créer des méthodes qui ne font pas d'effets de bord.